MatrixSDK/Crypto/CryptoMachine/MXCryptoMachine.swift

```diff
@@ -36,6 +36,8 @@ class MXCryptoMachine {
     }
     
     private static let kdfRounds: Int32 = 500_000
+    // Error type will be moved to rust sdk
```

Comment:
Maybe add this as a FIXME/TODO for visibility ?

Reply:
I am not a fan of adding [TODOs to code](https://www.osedea.com/en/blog/should-you-put-todos-in-the-source-code), but it is tracked in a draft issue on a sprint board so to be resolved asap